Sourcery/Models/TypeName.swift

```diff
@@ -26,27 +26,35 @@ protocol Typed {
 extension Typed {
     var isOptional: Bool { return typeName.isOptional }
     var unwrappedTypeName: String { return typeName.unwrappedTypeName }
+    var actualTypeName: TypeName? { return typeName.actualTypeName }
+    var isTuple: Bool { return typeName.isTuple }
 }
 
 // sourcery: skipDescription
 final class TypeName: NSObject, AutoDiffable {
     let name: String
 
+    /// Actual type name if given type name is type alias
```

Comment:
so how is this going to happen now? I think that before this PR TypeName was never used for typealiases